py/desiperf/instperfapp/pages/positioner.py

```diff
@@ -2,52 +2,79 @@
 from bokeh.layouts import column, layout
 from bokeh.models.widgets import Panel, Tabs
 from bokeh.models import ColumnDataSource
+from bokeh.models import Button, CheckboxButtonGroup, PreText, Select
 from bokeh.models.widgets.markups import Div
 from bokeh.plotting import figure
 import pandas as pd
+import numpy as np
 import glob, os
 
-from static.page import Page
+from static.plots import Plots
 
-class PosAccPage(Page):
-    def __init__(self, source):
-        self.page = Page('Positioner Accuracy Performance',source)
-        self.data_source = self.page.data_source
-        self.pos = str(3881)
+class PosAccPage():
+    def __init__(self, datahandler):
+        self.DH = datahandler
+        self.plots = Plots('Positioner Accuracy Performance',source=None)
+        self.data_source = self.plots.data_source
+        self.pos = str(1235)
         posfiles = glob.glob(os.getcwd()+'/instperfapp/data/per_fiber/*.csv')
         self.pos_list = [os.path.splitext(os.path.split(posf)[1])[0] for posf in posfiles] #np.linspace(0,4999,5000)
-        self.default_options = ['TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','OFFSET_0','OFFSET_FINAL']
-        self.pos_select = self.page.select(self.pos, self.pos_list)
-        self.x_select = self.page.select('OFFSET_0',self.default_options)
+        self.x_options = ['OFFSET_0','OFFSET_FINAL']
+        self.y_options = ['TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','EXPOSURE','total_move_sequences',
+                        'airmass','mirror_temp','truss_temp','air_mirror_temp_diff','wind_speed','wind_direction',
+                        'humidity','guide_meanxy','hex_rot_offset','ctrl_enabled']
+        self.pos_select = Select(value=self.pos, options=self.pos_list)
+        self.x_select = Select(value='OFFSET_0', options=self.x_options)
+        self.y_select = Select(value='TARGET_RA', options=self.y_options)
 
-        self.btn = self.page.button('Get Pos')
+        self.btn = Button(label = 'Get Pos', button_type='primary',width=300)
 
 
     def page_layout(self):
-        this_layout = layout([[self.page.header],
-                        [self.pos_select, self.x_select, self.btn],
-                        [self.ts1]])
-        tab = Panel(child=this_layout, title=self.page.title)
+        this_layout = layout([[self.plots.header],
```

Comment:
Upgrade suggestion: I think the scatter plot should go at the top (just below the pull-down menus), perhaps next to the correlation plot.